command.go

```diff
@@ -632,3 +692,47 @@ func (cmd *ScanCmd) parseReply(rd *bufio.Reader) error {
 
 	return nil
 }
+
+//------------------------------------------------------------------------------
+
+type ClusterSlotInfo struct {
+	Min, Max int
```

Comment:
Looking at http://redis.io/commands/cluster-slots I think it should be called `Start` and `End`